src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF2012.ProvideRuleProperties_Invalid.sarif

```diff
@@ -18,8 +18,8 @@
             },
             {
               "id": "TEST0002",
-              "name": "This isn't pascal case",
-              "helpUri": "http://docs.oasis-open.org/sarif/sarif/v2.1.0/sarif-v2.1.0.html"
+              "helpUri": "http://docs.oasis-open.org/sarif/sarif/v2.1.0/sarif-v2.1.0.html",
+              "name": "This isn't pascal case"
```

Comment:
test case logic is, read this json input into c# object, and write the object back to json, and then do analyze on it.

with the order current set, helpUri is before name, so the result will have the line number +1 from 21 to 22.
To make test more readable I changed this input to be the same order so when we see the error is line 22 we can understand which line it is pointing to. Let me know your thoughts.